### DISCUSSION TITLE
Is prisma making the DAL obsolete?

### DISCUSSION THREAD

**Discussion by moshe5745** [prisma/prisma#16510] [Q&A]
I am trying to figure out the best project structure with prisma.
And this question bothers me the most.

**Comment by nurul3101** [comment]
Hey @moshe5745 👋 
Can you please elaborate on your question?

Did you have a look at our [prisma-examples](https://github.com/prisma/prisma-examples) repository?
It should help you in structuring your prisma project.

**Reply by vladi-stevanovic** [reply]
👋 Hi @moshe5745, did you perhaps have a chance to check Nurul's questions? Let us know if you still need help!

Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
